x-pack/plugins/ml/server/models/data_recognizer/modules/metrics_ui_k8s/ml/k8s_memory_usage.json

```diff
@@ -0,0 +1,53 @@
+{
+    "job_type": "anomaly_detector",
+    "groups": [
+      "k8s",
+      "metrics"
+    ],
+    "description": "Metrics: Kubernetes - Identify unusual spikes in memory usage across kubernetes pods.",
+    "analysis_config": {
+      "bucket_span": "15m",
+      "detectors": [
+        {
+          "detector_description": "max('kubernetes.pod.memory.usage.node.pct') partitionfield='kubernetes.namespace'",
```

Comment:
As above, is it worth removing the `partitionfield` part of the description here? Otherwise the metrics UI must override this if it overrides the `partition_field_name` when creating the job.

Reply:
yep, removed in https://github.com/elastic/kibana/commit/5f013319cc34f6862ad17f9e7c27e0f1385da257